axelrod/tournament.py

```diff
@@ -376,4 +376,4 @@ def __init__(self, players, edges, match_generator=SpatialMatches,
 
         self.edges = edges
         self.match_generator = SpatialMatches(
-            players, turns, self.game, repetitions, edges)
+            players, turns, self.game, repetitions, self.noise, edges)
```

Comment:
https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/tests/unit/test_tournament.py#L595

`self.assertEqual(tournament.noise, 0.2)`

Should I add an extra test anyway or does this cover it ? I had this test written initial without realising I was not passing noise.